src/content/posts/4-way-handshake.md/index.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "4-Way Handshake - Wifi Authentication"
+summary: "In this article, you will learn how authentitcation works for WPA/WPA2 network"
+date: 2024-10-10T20:00:00+0200
+lastUpdate: 2024-10-10T20:00:00+0200
+tags: ["network", "wifi"]
+author: clementS
+draft: false
+---
+
+## Abstract
+To connect to a WiFi network, the authentication depends on the WiFi protocol. The most common protocols currently in use are 

```

Comment:
i would rather say a  WiFi based network. Is WiFi actually a protocol ? Wi-Fi is part of the IEEE 802 protocol family. (Wireless -Fidelity). Since you speak of multiple protocols, some precisions would be handy. 

Authentication depends on security protocols, the most common of which are: WEP, ... etc